src/Components/LabSwap/Selection.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react'
```

Comment:
Filename should start with the main component it's related to. ie. SwapSelection